### ISSUE TITLE
Add global config.json

### ISSUE THREAD

**Issue by jamesfancy** [tomoyukim/vscode-mermaid-editor#13]
Is there a way for global (user or system level) configuration? just like `.eslintrc`, such as 

- `~/.mermaid/config.json` in linux
- `%userprofile%\.mermaid\config.json` in windows

it's great to set `mermaid-editor.preview.defaultMermaidConfig` , but it does not support absolute path. I hope it can.

**Comment by jamesfancy** [comment]
BTW, it maybe a good idea to add a option `mermaid-editor.preview.theme` to set mermaid theme.

**Comment by jamesfancy** [comment]
My English is pool so I'd rather to use CODE to describe my idea.

```js
// from line 258 from Previewer.ts (Commit: badf3e440377c5c98ce0b7dea1c607b9c5dc3637)

const pathToDefaultConfig = Previewer.getConfiguration()
    .defaultMermaidConfig;
const workspaceFolders = vscode.workspace.workspaceFolders;
if (pathToDefaultConfig && workspaceFolders) {
    const uri = vscode.Uri.file(
        path.join(workspaceFolders[0].uri.fsPath, pathToDefaultConfig)
    );
    return await _readFile(uri);
} else {
    return JSON.stringify({
        theme: 'default'
    });
}
```

can be change to (just for refering)

```js
// restore theme setting
const { defaultMermaidConfig, theme } = Previewer.getConfiguration();

const workspaceFolders = vscode.workspace.workspaceFolders;
if (pathToDefaultConfig && workspaceFolders) {
    // check if it's a absolute path
    const pathToDefaultConfig = path.isAbsolute(defaultMermaidConfig)
        ? defaultMermaidConfig
        : path.join(workspaceFolders[0].uri.fsPath, pathToDefaultConfig);

    const uri = vscode.Uri.file(pathToDefaultConfig);
    return await _readFile(uri);
} else {
    return JSON.stringify({
        // if theme settings is valid, use it, otherwise 'default'.
        theme: theme ?? 'default'
    });
}
```

**Comment by tomoyukim** [comment]
Thank you for your proposal! That's a great suggestion but let me think about it.

**Comment by vintprox** [comment]
I'd say current default theme of mermaid is itself a reason to drop it, but some assistance from plugin also won't hurt: as author of this issue asks, please add theme choice VSC setting. Of course, it's a convenience that even live editor lacks (which just exposes plain config object), but I'm sure dropdown picker can improve UX. Thanks in advance!

**Comment by tomoyukim** [comment]
I released v0.7.0 with supporting absolute path and reverting `mermaid-editor.preview.theme` config. Please check it out and. Thank you.
